get_strings.py

```diff
@@ -0,0 +1,62 @@
+from json import dumps
```

Comment:
Is the repo root the best location for this utility?

Reply:
It was more that I didn't really know of a better place.  Could create a folder for it if need be